### PR TITLE
Feature/login page

### PR DESCRIPTION
**Pull request by hogmann**

### CODE REVIEW

File: prime-angular-frontend/src/assets/pill-1.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="1272px" height="608px" viewBox="0 0 1272 608" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 61.2 (89653) - https://sketch.com -->
+    <title>pill 1</title>
+    <desc>Created with Sketch.</desc>
```
Comment (itchison) [inline prime-angular-frontend/src/assets/pill-1.svg]: Should this be here?

File: prime-angular-frontend/src/app/modules/auth/pages/info/info.component.html [hunk 2]
```diff
@@ -1,42 +1,101 @@
...
+        </span>
+        <span class="lead ml-2">(for prescribers only)</span>
+        <div class="mt-4"><button mat-flat-button
+                  class="mr-4"
                   color="primary"
-                  (click)="loginUsingBCSC()">Log-in with your BC Services Card</button>
+                  (click)="loginUsingBCSC()">Log in with BC Services Card</button>
+          <button mat-flat-button
+                  color="accent"
+                  href="#">Learn More</button></div>
```
Comment (itchison) [inline prime-angular-frontend/src/app/modules/auth/pages/info/info.component.html]: Do we have any information on where this button should go?

File: prime-angular-frontend/src/assets/pill-1.svg (hunk 1 above)
Comment (hogmann) [reply prime-angular-frontend/src/assets/pill-1.svg]: Can pull this out of the svg, as I assume we don't want this information there.

**Comment by hogmann** [comment]
@mtpultz not sure what is happening with this PR. Will look into this tomorrow morning since all the changes are done to be ready for the demo's for monday.

File: prime-angular-frontend/src/app/modules/auth/pages/info/info.component.html [hunk 3]
```diff
@@ -42,25 +42,37 @@
...
+            card on your mobile device.</p>
           <a href=""><small class="anchor font-weight-bold">SET UP MY BC SERVICES CARD</small></a>
         </div>
         <div class="col-sm-12 col-md-3 info mb-4">
           <h3>PRIME Enrolment</h3>
-          <p>Provide contact and Health Professional information. Answer a set of self-declaration questions, then
-            carefully read and accept the PharmaNet terms of access.</p>
+          <p>Provide contact and Health
+            Professional information.
+            Answer a set of selfdeclaration
```
Comment (itchison) [inline prime-angular-frontend/src/app/modules/auth/pages/info/info.component.html]: lost the dash here.

**Comment by mtpultz** [comment]
@hogmann looks like the most recent merge into this branch will need to be backed out.

File: prime-angular-frontend/src/app/modules/auth/pages/info/info.component.html [hunk 4]
```diff
@@ -1,42 +1,117 @@
...
-      <div class="d-flex flex-column align-items-center">
-
-        <img src="/assets/prime_logo.svg"
+<div>
+  <div class="container-fluid">
+    <div class="row mt-4 enrolment-container">
+      <div class="col-md-3 d-none d-md-block">
+        <img src="/assets/prime-logo-textless.svg"
+             height="200"
+             width="200"
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/auth/pages/info/info.component.html]: We'll probably want these dimensions in the component stylesheet

File: prime-angular-frontend/src/app/modules/auth/pages/info/info.component.html (hunk 2 above)
Comment (hogmann) [reply prime-angular-frontend/src/app/modules/auth/pages/info/info.component.html]: Currently we do not. Will make a ticket to note these need to be linked up.

**Pull request merged**